examples/README.md

```diff
@@ -0,0 +1,34 @@
+# Examples
+
+We provide a variety of examples for deep learning frameworks including [PyTorch](https://github.com/huggingface/transformers/tree/main/examples/pytorch), [JAX](https://jax.readthedocs.io/en/latest/), and [PaddlePaddle](https://github.com/PaddlePaddle/Paddle). 
```

Comment:
The 3 framework links all lead to different things:
1. PyTorch leads to HF PyTorch examples.
2. Jax leads to its docs
3. And PaddlePaddle leads to the code.
Let's make it consistent.

Reply:
I'll stick to github repos